code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2632,3 +2632,26 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	if(islizard(M))
 		SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "triumph", /datum/mood_event/memories_of_home, name)
 	..()
+
+/datum/reagent/consumable/ethanol/the_juice
+	name = "The Juice"
+	description = "Woah man, this like, feels familiar to you dude."
+	color = "#4c14be"
+	boozepwr = 50
+	quality = DRINK_GOOD
+	taste_description = "like, the future, man"
+	var/datum/brain_trauma/special/bluespace_prophet/J
```

Comment:
new variables should be at the end of the definition, put it below `chemical_flags`

Reply:
Also, use a more descriptive variable name than `J`. maybe... `prophet_trauma`?

Reply:
Drink_good.